src/DssSpell.t.sol

```diff
@@ -2231,11 +2220,7 @@ contract DssSpellTest is DSTest, DSMath {
 
         ChainlogAbstract chainLog = ChainlogAbstract(addr.addr("CHANGELOG"));
 
-        assertEq(chainLog.getAddress("GUNIV3DAIUSDC1"), addr.addr("GUNIV3DAIUSDC1"));
-        assertEq(chainLog.getAddress("PIP_GUNIV3DAIUSDC1"), addr.addr("PIP_GUNIV3DAIUSDC1"));
-        assertEq(chainLog.getAddress("MCD_JOIN_GUNIV3DAIUSDC1_A"), addr.addr("MCD_JOIN_GUNIV3DAIUSDC1_A"));
-        assertEq(chainLog.getAddress("MCD_CLIP_GUNIV3DAIUSDC1_A"), addr.addr("MCD_CLIP_GUNIV3DAIUSDC1_A"));
-        assertEq(chainLog.getAddress("MCD_CLIP_CALC_GUNIV3DAIUSDC1_A"), addr.addr("MCD_CLIP_CALC_GUNIV3DAIUSDC1_A"));
+        // Insert new changelog values tests here
```

Comment:
I'd suggest commenting this test when not used. It's more time for running the tests/CI + it throws a warning do the unused variable.

Reply:
Done in cf225e4.